src/components/Messages/ChatMenu.vue

```diff
@@ -89,6 +89,7 @@
     methods: {
       conversationSelected(name, username, img, messages) {
         this.$emit("clicked", name, username, img, messages);
```

Comment:
interesting concept :thinking:

Reply:
Yeah it's because I made this component which is a child component (the chat menu to select a conversation) but since its a child component, the parent (chat window) needs to see which chat was selected, hence the $emit